ec/acc/deployment_basic_defaults_test.go

```diff
@@ -164,3 +168,15 @@ func TestAccDeployment_basic_defaults(t *testing.T) {
 		},
 	})
 }
+
+func testAccDeploymentResourceBasicDefaults(t *testing.T, fileName, name, region, depTpl string) string {
```

Comment:
These functions are called since their name starts with `test...`, If we change them to be called `fixture...` no need to add the `acceptance` tag to a ll the files and Make target.

Reply:
🤦‍♀️ 😄

Reply:
```
--- FAIL: TestAccDeployment_basic (0.00s)
Error:     deployment_basic_test.go:159: authwriter: 1 error occurred:
        	* one of apikey or username and password must be specified
```

Looks like we do need to skip building these tests after all

Reply:
yeah I you're right, since the tests themselves start with `Test....